### DISCUSSION TITLE
I did webinstall.dev and webi in Caddy Documentation.

### DISCUSSION THREAD

**Discussion by fu-sen** [webinstall/webi-installers#242] [General]
have been using Caddy for some time. I recently looked at the docs and learned about webinstall.dev and webi.
https://caddyserver.com/docs/install#webi

I'm running a server on Windows. I used to manage Caddy with [Scoop](https://scoop.sh/),
but automatic update is not suitable for running Caddy. I also wanted to try the beta version of Caddy.
webi met that, so I switched to webi.

___

[Glitch](https://glitch.com/) project with Caddy on webinstall.dev or webi:
https://caddy.community/t/use-caddy-with-glitch-glitch-com-glitch-me-information-as-of-may-2021/12434

I used to download Caddy directly from GitHub release. Using webi simplifies this command.

**Comment by coolaj86** [comment]
Thanks!
